javascript/lib/api/src/auth/bearer-auth.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
```

Comment:
The year of the license header should be the year when the file was created.

Reply:
I tried that but tslint immediately started comlaining about a missing file headers... I'll fix it in both the config and this file, then.